MekHQ/src/mekhq/gui/utilities/Java2sAutoComboBox.java

```diff
@@ -0,0 +1,82 @@
+package mekhq.gui.utilities;
```

Comment:
Copyright header.

Reply:
This one is weird, as I lifted it straight from stack overflow without even pretending to modify it. Not sure how that should go re: copyright header.

Reply:
Yeah, turns out I'll be removing this - it doesn't really do what I need it to do and my implementation of the relevant code went in a different direction.